### ISSUE TITLE
Will dragonfly work with Windows 10 Speech Recognition?

### ISSUE THREAD

**Issue by astrotiff** [t4ngo/dragonfly#68]
Hello, I was wondering if dragonfly is compatible with Windows 10 since the documentation does not currently mention this OS. I understand that the package must be used with Python v 2.x. I appreciate any input.

Thank you!

**Comment by tylercal** [comment]
Yes. I’ve been using it on windows 10 without issues since the release.
